src/library-authoring/common/context.tsx

```diff
@@ -46,98 +62,136 @@ export interface LibraryContextData {
  */
 const LibraryContext = React.createContext<LibraryContextData | undefined>(undefined);
 
+interface LibraryProviderProps {
+  children?: React.ReactNode;
+  libraryId: string;
+  collectionId?: string;
+  componentPickerMode?: boolean;
+  parentLocator?: string;
```

Comment:
Please add a `/** JSDoc comment */` explaining what `parentLocator` is for. And `componentPickerMode` too :)

Reply:
Here: ded2f077ec24500e65a709a2276980d3977d8bf1